beetmoverscript/utils.py

```diff
@@ -175,17 +175,21 @@ def generate_beetmover_template_args(context):
 def _generate_beetmover_template_args_maven(task, release_props):
     tmpl_args = {
         'artifact_id': task['payload']['artifact_id'],
-        'branch': release_props['branch'],
-        'buildid': release_props['buildid'],
+        'branch': release_props.get('branch'),
```

Comment:
I think we should strip `branch` and `buildid` from `tmpl_args`. This way, if the template ever refer to them, we would know it compilation time. Otherwise, someone has to check the results manually.

Reply:
Good point, done. However, for the sake of cleaning, if `beetmover-geckoview` jobs aren't using these arguments either, we should strip them in-tree as well.

Reply:
Good point!